### ISSUE TITLE
"workingDir" from goggame.info is not used

### ISSUE THREAD

**Issue by Zauberfisch** [sharkwouter/minigalaxy#158]
The `goggame-\d+.info` file of some games contains `"workingDir": "..."` to indicate the binary is to be executed in a certain directory.

An example game is `The Witcher 3: Wild Hunt - Game of the Year Edition`
```
{
    "clientId": "49292044087338605", 
    "gameId": "1495134320", 
    "language": "English", 
    "languages": [
        "en"
    ], 
    "name": "The Witcher 3: Wild Hunt - Game of the Year Edition", 
    "overlaySupported": true, 
    "playTasks": [
        {
            "category": "game", 
            "isPrimary": true, 
            "languages": [
                "*"
            ], 
            "name": "The Witcher 3: Wild Hunt - Game of the Year Edition", 
            "path": "bin//x64//witcher3.exe", 
            "type": "FileTask", 
            "workingDir": "bin//x64"
        }, 
        {
            "languages": [
                "*"
            ], 
            "link": "http://www.gog.com/en/support/the_witcher_3_wild_hunt", 
            "name": "Support", 
            "type": "URLTask"
        }
    ], 
    "rootGameId": "1495134320", 
    "version": 1
}
```

Starting the game:

* **ERROR** Starting through minigalaxy:
```
Failed to start The Witcher 3: Wild Hunt - Game of the Year Edition:
00c0:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00c8:fixme:ver:GetCurrentPackageId (000000000363FDA0 0000000000000000): stub
00c0:fixme:msvcrt:__clean_type_info_names_internal (0x2411878) stub
00c0:fixme:msvcrt:__clean_type_info_names_internal (0x2325768) stub
```
* **ERROR** `cd "/home/$USER/GOG Games/Witcher3"; WINEPREFIX="/home/$USER/GOG Games/Witcher3/prefix" wine /home/$USER/GOG\ Games/Witcher3/bin/x64/witcher3.exe`
```
00c0:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00c8:fixme:ver:GetCurrentPackageId (000000000332FDA0 0000000000000000): stub
00c0:fixme:msvcrt:__clean_type_info_names_internal (0x2411878) stub
00c0:fixme:msvcrt:__clean_type_info_names_internal (0x2325768) stub
```
* **SUCESS** `cd "/home/$USER/GOG Games/Witcher3/bin/x64"; WINEPREFIX="/home/$USER/GOG Games/Witcher3/prefix" wine /home/$USER/GOG\ Games/Witcher3/bin/x64/witcher3.exe`
Game starts successfully

For me, the game will only start if I execute `witcher3.exe` inside `~/GOG\ Games/Witcher3/bin/x64`.

**Comment by sharkwouter** [comment]
Thanks for reporting this, I wasn't aware of this file. That's really useful

**Comment by sharkwouter** [comment]
A pull request has been merged which should solve this issue. Could you clone the master branch and report back?

**Comment by sharkwouter** [comment]
This should be solved since Minigalaxy 1.0.0
